File: cpp/problem453.cpp
class Solution {
public:
    int minMoves(vector<int>& nums) {
        int sum = 0, min_val = nums[0];
        for(auto num: nums){
            sum += num;
            if( num < min_val ) min_val = num;
        }
        return sum - min_val*nums.size();
    }
};

File: python/problem453.py
class Solution:
    def minMoves(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        return sum(nums)-len(nums)*min(nums)
        
        
# k*(n-1)+sum(nums)=n*min(nums)+k*n  
# n=len(nums)  k is answer
